### ISSUE TITLE
[4.0 Alpha 15] Texture BUG? on specific models.  

### ISSUE THREAD

**Issue by decryptedchaos** [godotengine/godot#65379]
### Godot version

4.0 Alpha 15

### System information

Windows 10  GTX 1080

### Issue description

Odd behavior in alpha 15.   some models have odd texture on them.   as show in the below example.  this model was textured in blender.   it is meant to be a shiny metal material.   Which was working up until Alpha 14.  

On loading the project in Alpha 15 this is what occurs. 

![weirdtexture](https://user-images.githubusercontent.com/1302391/188508859-160cb269-7290-40f9-8a9f-af9db01646f1.png)

All the wavy texture is **not** the expected result.  I read there was a normal issue in alpha 15.  Could this be related, the PR wasn't very self-explanatory.  We debated if this means mesh normals or texture normals in discord.  Still unclear.  I decided to leave the reference in case it is related. 

![gdcl](https://user-images.githubusercontent.com/1302391/188512654-066aa5a2-a745-44bf-9a73-921cb122af39.png)

Also, furthermore, note some inconsistency in how the editor is parsing and displaying the texture.  

![ic1](https://user-images.githubusercontent.com/1302391/188508520-c174df13-d912-4105-916c-6d4c940838bc.png)

Overall Model shows with the incorrect texture.  

![ic2](https://user-images.githubusercontent.com/1302391/188508553-78bc9993-f675-44c6-9d02-9114b93eb682.png)

Material preview shows with a checker pattern (shouldn't be there) but the large material preview shows the texture as it should be. 

**I should note, once a project/asset is opened in alpha 15 this seems to break the project in this way, even opening in alpha 14 after it has been opened in alpha 15 the issue will remain.**

### Steps to reproduce

This only happens on specific models.   I haven't found the pattern yet for which models are causing it.  but it is happening none the less and very specifically only after being opened in alpha 15

**Comment by Omgesus** [comment]
My issue is similar to this one, but the opposite. The materials are intended to be dull (or rather dull), but have an extreme amount of specular reflections to it now, along with the same "splotchy" appearance. So, the same issue, but with the opposite intended material type (Chaos' material was intended to be specular/shiny, mine weren't). This also occurs on a few other materials in the project, but may be due to a different issue. 

Issue occurs on Linux (Manjaro 21.3.7). Alpha 15 is the first buggy screenshot, while Alpha 14 (on a backup of the project from August 30th) has no issue.

![2022-09-05-121942_1899x2094_scrot](https://user-images.githubusercontent.com/22787327/188510473-c7a32ff1-aadf-4aab-af3e-4bbcd2cd9e53.png)
![2022-09-05-123643_1899x2094_scrot](https://user-images.githubusercontent.com/22787327/188510476-ad99b4a0-c58d-43b3-b3a2-0341b5fc9906.png)

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot/issues/64854.
